doc/api/esm.md

```diff
@@ -1507,7 +1507,7 @@ console.log(VERSION);
 With this loader, running:
 
 ```console
```

Comment:
```suggestion
```bash
```

`console` is only supposed to be used in conjunction w/ leading shell dollar signs (`$`) and definitely not for one-line commands w/ no output.

Reply:
2246291

Reply:
I don't think we want `bash` either because that will do bash syntax highlighting. I think we want `text` here. But there's probably lots of instances of that to fix in our docs, so I'll do that in a subsequent PR. Just an FYI.